release/create_release_archives.sh

```diff
@@ -76,7 +76,7 @@ function create_windows_archive() {
 
 pushd ${ROOT}
 cp istio.VERSION "${COMMON_FILES_DIR}"/
-find samples install -type f -name "*.yaml" \
+find samples install -type f -name "*.yaml" -o -name "cleanup*" \
```

Comment:
Just ran this, this does not work, you need to do:
\\( -name "*.yaml" -o -name "cleanup*" \\)

also when I ran with my change the only file added is:
samples/apps/bookinfo/cleanup.sh

Is that what you expect ?